samples/hello-world-expo/src/SecondPanel.tsx

```diff
@@ -0,0 +1,158 @@
+
+interface SecondPanelProps {
+    onNavigateBack: () => void;
+}
+
+interface SecondPanelState {
+    toggleValue?: boolean;
+    progressValue?: number;
+}
+
+const styles = {
+    scroll: RX.Styles.createScrollViewStyle({
+        alignSelf: 'stretch',
+        backgroundColor: '#f5fcff'
+    }),
+    container: RX.Styles.createViewStyle({
+        padding: 16,
+        justifyContent: 'center',
+        alignItems: 'center'
+    }),
+    titleText: RX.Styles.createTextStyle({
+        fontSize: 16,
+        textAlign: 'center',
+        marginTop: 12,
+        color: 'black'
+    }),
+    videoTitleText: RX.Styles.createTextStyle({
+        marginBottom: 8
+    }),
+    progressMargin: RX.Styles.createViewStyle({
+        margin: 8
+    }),
+    video: RX.Styles.createViewStyle({
+        height: 176,
+        width: 320
+    }),
+    roundButton: RX.Styles.createViewStyle({
+        margin: 16,
+        borderRadius: 16,
+        backgroundColor: '#7d88a9'
+    }),
+    buttonText: RX.Styles.createTextStyle({
+        fontSize: 16,
+        marginVertical: 6,
+        marginHorizontal: 12,
+        color: 'white'
+    })
+};
+
+class SecondPanel extends RX.Component<SecondPanelProps, SecondPanelState> {
+    private _progressTimerToken: number;
+
+    constructor() {
+        super();
+
+        this.state = {
+            toggleValue: true,
+            progressValue: 0
+        };
+    }
+
+    componentDidMount() {
+        this._startProgressIndicator();
+    }
+
+    componentWillUnmount() {
+        this._stopProgressIndicator();
+    }
+
+    render() {
+        return (
+            <RX.ScrollView style={ styles.scroll }>
+                <RX.View style={ styles.container }>
+                    <RX.Button style={ styles.roundButton } onPress={ this._onPressBack }>
+                        <RX.Text style={ styles.buttonText }>
+                            Go Back
+                        </RX.Text>
+                    </RX.Button>
+
+                    <RX.Text style={ styles.titleText }>
+                        Here is a simple control built using ReactXP
+                    </RX.Text>
+                    <ToggleSwitch
+                        value={ this.state.toggleValue }
+                        onChange={ this._onChangeToggle }
+                    />
+
+                    <RX.Text style={ styles.titleText }>
+                        Here is an SVG image using the ImageSvg extension
+                    </RX.Text>
+                    {/* <ProgressIndicator
+                        style={ styles.progressMargin }
+                        progress={ this.state.progressValue }
+                        fillColor={ '#ddd' }
+                        size={ 32 }
+                    /> */}
+
+                    <RX.Text style={ [styles.titleText, styles.videoTitleText] }>
+                        Here is a video using the Video extension
+                    </RX.Text>
+                    {/* <RXVideo

```

Comment:
I recently fixed video package.
If it was your main concern here - you could fix the PR accordingly.
See: https://github.com/Microsoft/reactxp/pull/229
for reference.

Reply:
Done!